DataMonitoring/GroundStationWaterflow.py

```diff
@@ -180,8 +162,14 @@ def process():
                 display = True
             elif c == 'f':
                 display = False
+            elif c == '4':
```

Comment:
Instead of 4, let's do 'd' as the key to toggle the display mode. Because we might later change the number of data points.